### DISCUSSION TITLE
doris1.0.0 be 在查询数据中挂掉

### DISCUSSION THREAD

**Discussion by csj-9-9** [apache/doris#8999] [A - General / Q&A]
select bitmap_count(bitmap_intersect(b))
from
(
select dt, bitmap_union(to_bitmap(user_id)) b from (
select 1 dt, 2 user_id
from bitmap_test a
where 1=0
) a
group by a.dt
) t2;

执行上述sql后，be进程挂掉。

doris版本：1.0.0

使用gdb查看到的异常：

#0 cardinality (this=) at /mnt/disk1/cmy/git/incubator-doris/be/src/util/bitmap_value.h:1376
1376 /mnt/disk1/cmy/git/incubator-doris/be/src/util/bitmap_value.h: No such file or directory.

 
各位大佬，为什么会出现这个问题？应该怎么解决呢？

**Comment by csj-9-9** [comment]
![image](https://user-images.githubusercontent.com/9983962/163108369-6e7527c4-0773-4c2b-8d02-58061ffbb86b.png)

**Comment by dataalive** [comment]
具体的版本号是？可以尝试下dev-1.0.1 branch。

**Comment by spihiker** [comment]
大佬。看下9279。扩容增加be和fe节点时。也是fe leader直接挂掉。
